CONTRIBUTING.md

```diff
@@ -14,13 +14,8 @@ Just as any language that is written in itself, LALRPOP has to solve chicken-and
 To compile LALRPOP we need a working version of LALRPOP. To have a working version we need
 to compile it first.
 
-In the past, LALRPOP used snapshots of its older self to compile itself (you might notice
-[lalrpop-snap](https://crates.io/crates/lalrpop-snap) on crates.io). Unfortunately, this meant end-users
-had to build **both** `lalrpop` and `lalrpop-snap` crates and nobody liked this double-building.
-This approach has been abandoned a while ago.
-
-Today's approach is different - `lalrpop` crate includes already auto-generated parser
-in `lalrpop/src/parser/lrgrammar.rs` that's end-users use.
+`lalrpop` crate includes already auto-generated parser
```

Comment:
(Unrelated to this PR, but while we're at it). Or just `lalrpop` ? but "`lalrpop` crate" sounds slightly odd to me.
 
```suggestion
The `lalrpop` crate already includes an auto-generated parser
```

Reply:
"`lalrpop` crate" sounds better. Because it is not true for `lalrpop-util`